pool.go

```diff
@@ -10,15 +10,18 @@ type PoolService service
 type PoolComponent struct {
 	TokenAddress string `json:"token_address"`
 	Type         string `json:"type"`
+	Token        Token  `json:"token"`
```

Comment:
Token có được tự động load không b @bluevariant 
Với mục đích thêm ở đây lmj nhỉ vì mình đang bỏ đi để hạn chế kích thước của response quá lớn

Reply:
Mình có thể thêm options để load token hoặc k để giảm response. Mình đang cần load thêm token để tương thích vs code repo ở bên backend: https://github.com/depocket/depocket_backend_monorepo/blob/3651b9f0579716da2e4ecf330d7c935b1d915a61/core/repositories/persistence/pool.go#L46

Reply:
Uhm, vậy mình thấy ko có vde
PR này không quá to nên mình sẽ skip test response ở local nên bạn check kĩ xem work sao nhé
----

> thêm options để load token hoặc k

Phần này tạo issue làm sau nhé, vì độ ưu tiên cũng chưa cao

Reply:
Oki b, mình test local ok r nhé

Reply:
nhưng mình đang thử integrate bên backend xem có work k